meshapp/src/components/profile/profile-textfield.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
```

Comment:
I know this is a component, but I don't think profile-textfield would be used anywhere else so it might make more sense to have it in /src/profile/.